packages/notifications-flyout/src/__fixtures__/createSampleNotifications.js

```diff
@@ -7,7 +7,6 @@ import Timestamp from "@hig/timestamp";
 import avatarImagePath from "@hig/storybook/storybook-support/fixtures/avatar/chris-reynolds.png";
 import thumbnailImagePath from "@hig/storybook/storybook-support/fixtures/thumbnail/greenhouse.png";
 import "@hig/avatar/build/index.css";
-import "@hig/text-link/build/index.css";
```

Comment:
This is a bit weird since `text-link` isn't actually in the `package.json` in both `notifications-flyout` and `rich-text`. Wondering if it might be better to add `text-link` as a `devDependency` pointing to the current version, and you bump up `text-link` to the next major version since there are breaking changes. 

But also not sure if that would make sense to do all that for stuff that shows up in `storybook` or `gemini` testing. With that being said I think the commit types should be `docs:` or `test:` depending on where the change lives as I don't think we need to see a version bump on the packages.